content/blog/2017/10/servicenow-automation.md

```diff
@@ -0,0 +1,114 @@
++++
+author = "John Schoewe"
+author_url = "https://github.com/jschoewe"
+categories = ["John Schoewe", "Encore"]
+date = "2017-10-19"
+description = "How to use Python and the ServiceNow API to Automatically Create and Update Configuration Items"
+featured = ""
+featuredalt = ""
+featuredpath = ""
+linktitle = ""
+title = "Automated Editing and Addition of ServiceNow CIs"
+type = "post"
+
++++
+
+The following is a quick tutorial on using the ServiceNow API with python to programatically create and update configuration items. A personal development instance of ServiceNow Jakarta was used for testing the code below.
+
+<br />
+# About the ServiceNow API
+
+Servicenow uses a REST Table API that allows users to perform create, read, update and delete operations on existing tables. ServiceNow also provides you with a REST API Explorer that uses information from your instance to provide you with a list of endpoints, methods, and variables that you can use to build and send a REST request. This explorer can be found by navigating to 'System Web Services' -> 'REST API Explorer'.
+
+NOTE: Before accessing the API, be sure your user role is either web_service_admin, rest_api_explorer, or admin.
```

Comment:
Maybe provide steps on how to change the role, or a link to documentation how to do it?